frontend/webservice/interface.cgi

```diff
@@ -188,9 +205,25 @@ sub get_workgroup_interfaces{
     }
 
     return {results => \@res};
+}
 
-
-    
+sub get_interfaces{
+    my $method = shift;
+    my $params = shift;
+    my $node_id = $params->{'node_id'}{'value'};
+    my $workgroup_id = $params->{'workgroup_id'}{'value'};
+    my $interfaces;
+    if (defined $workgroup_id){
+        $interfaces = OESS::DB::Interface::get_interfaces(db => $db, node_id => $node_id);
+    } else {
+        $interfaces = OESS::DB::Interface::get_interfaces(db => $db, node_id => $node_id);
+    }
```

Comment:
I'm guessing you meant to pass workgroup_id into get_interfaces if it was defined? You really don't need to; If an arg is undef it won't be filtered on, so you can go ahead and pass node_id and workgroup_id into get_interfaces regardless.